exchanges/bitstamp/bitstamp_test.go

```diff
@@ -500,174 +518,161 @@ func TestSubmitOrder(t *testing.T) {
-		t.Error("Expecting an error until QA pass is completed")
+	})
+	if !mockTests {
+		assert.ErrorContains(t, err, "You have only 0 USD available. Check your account balance for details.")
+	} else {
+		assert.NoError(t, err, "SubmitOrder should not error")
+		assert.Equal(t, float64(45), o.Amount, "Amount should be correct")
+		assert.Equal(t, asset.Spot, o.AssetType, "AssetType should be correct")
+		assert.Equal(t, "123456789", o.ClientID, "ClientID should be correct")
+		assert.Equal(t, "1234123412341234", o.OrderID, "OrderID should be correct")
+		assert.Equal(t, 2211.0, o.Price, "Price should be correct")
+		assert.Equal(t, btcusdPair, o.Pair, "Pair should be correct")
+		assert.WithinRange(t, o.Date, time.Now().Add(-24*time.Hour), time.Now(), "Date should be correct")
 	}
 }
 
 func TestCancelExchangeOrder(t *testing.T) {
 	t.Parallel()
 
 	if !mockTests {
-		sharedtestvalues.SkipTestIfCannotManipulateOrders(t, b, canManipulateRealOrders)
-	}
-
-	orderCancellation := &order.Cancel{
-		OrderID:   "1234",
-		AssetType: asset.Spot,
+		sharedtestvalues.SkipTestIfCredentialsUnset(t, b, canManipulateRealOrders)
 	}
-	err := b.CancelOrder(context.Background(), orderCancellation)
-	switch {
-	case !sharedtestvalues.AreAPICredentialsSet(b) && err == nil && !mockTests:
-		t.Error("Expecting an error when no keys are set")
-	case sharedtestvalues.AreAPICredentialsSet(b) && err != nil && !mockTests:
-		t.Errorf("Could not cancel orders: %v", err)
-	case mockTests && err == nil:
-		t.Error("Expecting an error until QA pass is completed")
+	err := b.CancelOrder(context.Background(), &order.Cancel{
+		OrderID: "1453282316578816",
+	})
+	if !mockTests {
+		assert.ErrorContains(t, err, "Order not found")
+	} else {
+		assert.NoError(t, err, "CancelExchangeOrder should not error")
 	}
 }
 
 func TestCancelAllExchangeOrders(t *testing.T) {
 	t.Parallel()
 
 	if !mockTests {
-		sharedtestvalues.SkipTestIfCannotManipulateOrders(t, b, canManipulateRealOrders)
+		sharedtestvalues.SkipTestIfCredentialsUnset(t, b, canManipulateRealOrders)
 	}
-
 	resp, err := b.CancelAllOrders(context.Background(),
 		&order.Cancel{AssetType: asset.Spot})
-	switch {
-	case !sharedtestvalues.AreAPICredentialsSet(b) && err == nil && !mockTests:
-		t.Error("Expecting an error when no keys are set")
-	case sharedtestvalues.AreAPICredentialsSet(b) && err != nil && !mockTests:
-		t.Errorf("Could not cancel orders: %v", err)
-	case mockTests && err != nil:
-		t.Errorf("Could not cancel orders: %v", err)
-	}
-
+	assert.NoError(t, err, "TestCancelAllExchangeOrders should not error")
 	if len(resp.Status) > 0 {
 		t.Errorf("%v orders failed to cancel", len(resp.Status))
 	}
 }
 
 func TestModifyOrder(t *testing.T) {
 	t.Parallel()
-	sharedtestvalues.SkipTestIfCannotManipulateOrders(t, b, canManipulateRealOrders)
-	_, err := b.ModifyOrder(context.Background(), &order.Modify{AssetType: asset.Spot})
-	if err == nil {
-		t.Error("ModifyOrder() Expected error")
+
+	if !mockTests {
```

Comment:
Can rid this check since the wrapper func isn't supported

Reply:
done